ts/features/idpay/details/components/TimelineOperationListItem.tsx

```diff
@@ -167,17 +171,21 @@ const getTransactionOperationProps = (
       paymentLogoIcon,
       title,
       subtitle,
-      transactionStatus: "reversal",
-      badgeText: getBadgeTextByTransactionStatus("reversal")
+      transaction: {
+        badge: getBadgePropsByTransactionStatus("reversal")
+      }
     };
   }
 
   return {
     paymentLogoIcon,
     title,
     subtitle,
-    transactionStatus: "success",
-    transactionAmount: getAccruedString()
+    transaction: {
+      amount: getAccruedString(),
+      amountAccessibilityLabel:
+        getAccessibleAmountText(getAccruedString()) ?? getAccruedString()
```

Comment:
Since `getAccruedString` returns a string in any case, is the fallback operation necessary?

Reply:
The `getAccessibleAmountText` utility function could return an `undefined` in its definition, so it means that this fallback operation is necessary.